DecipherNow/gm-ui-components
# feat: controllable collapse component

Resolves #504

changes
- implement control props and update tests
- change the icon mock from `IconMock` to `icon-mock`. This is necessary because I have to fully mount the collapse component in the test due to hooks, and it gave me all sorts of errors about improper casing.

to test
- run storybook and make sure the controlled and default components work as expected
